shared/ui-components/src/page_components/listing/ListingsList.tsx

```diff
@@ -30,11 +30,16 @@ const ListingsList = (props: ListingsProps) => {
       )
     }
 
+    // address as subtitle
+    const { street, city, state, zipCode } = listing?.buildingAddress || {}
+    const subtitle = `${street}, ${city} ${state}, ${zipCode}`
```

Comment:
I think the comma here should be between city and state rather than state and zipcode, but I want to defer to @slowbot before changing it.